### ISSUE TITLE
gtfs_schedule_history_load cannot parse Tulare County Regional's feed data

### ISSUE THREAD

**Issue by machow** [cal-itp/data-infra#691]
![image](https://user-images.githubusercontent.com/2574498/141169509-8ebda187-f90c-4243-a384-b3f10b6f4124.png)

https://github.com/cal-itp/data-infra/blob/main/airflow/data/agencies.yml#L1492

**Comment by machow** [comment]
Looks like a misformatted row (note the unescaped text with quote marks):

```
C40,Weekday,t_500090_b_17398_tn_0,Rt 40 Southbound,Southbound,0,110879,p_499962,,,,3.0t+5.00,2.0t+3.00,1440.00,Call 1-800-431-9711 at least one day in advance to request a deviation for pickup. Available for ADA-eligible riders.,Call 1-800-431-9711 at least one day in advance to request a deviation for drop off. Available for ADA-eligible riders.,"Wave or flag down the bus at a safe pullout location. For your convenience, you may call a day in advance to let us know you will be waiting for the bus somewhere other than a bus stop.",Request drop off and the driver will stop where it is safe to do so.

```

**Comment by evansiroky** [comment]
Fixed in #716
